trade-finance-manager-ui/templates/utilisation-reports/find-utilisation-reports-by-year.njk

```diff
@@ -87,5 +88,5 @@
 {% block sub_content %}{% endblock %}
 
 {% block page_scripts %}
-  <script src="/assets/js/enableFindReportsByYearDropdown.js" integrity="sha512-QLvf2zIp3WAaUOyTIf4SlZ9JLBWVw1CCwwY8Ozzc3+mTsFv4WClwZ9ZYNdNwf9nSFNd2Y4WNr1GL/Bc6ABo1Fw==" type="text/javascript" crossorigin="anonymous"></script>
+  <script src="/assets/js/enableFindReportsByYearDropdown.js" integrity="EFJiI4d7Q8VRfgYiiboT3GIz93joREbGYasuLLcdGROS7srBMp+Y+b9b5U389AXQhsDbAkuSwnZtEmnRUWrYSg==" type="text/javascript" crossorigin="anonymous"></script>
```

Comment:
If not already, can you check the other script's integrity hashes? `mojFilterHide` and `ssoRedirectAutoSubmit have been re-generated

Reply:
updated for mojFilterHide.  Cannot find any integrity hashes for ssoRedirectAutoSubmit

Reply:
@Zainzzkk it's missings the `sha512-` prefix?

Reply:
added back